README.md

```diff
@@ -43,6 +43,14 @@ $ make build
 
 Add the `bin` folder to your `$PATH` or copy the `bin/pulumi-resource-aws-native` file to another location in your `$PATH`.
 
+### Running tests
+
+To run unittests, use:
+
+```
+$ make test_provider
```

Comment:
[nit] this could be `inline` like in the other added paragraph to keep it compact